tasks/main.yml

```diff
@@ -1,13 +1,15 @@
 ---
+- name: Set default variables
+  include_vars: default.yml
 
 - name: Set version specific variables
-  include_vars: "{{ item }}"
+  include_vars: "{{ item | replace(' ','-') }}"
```

Comment:
interesting - why is this needed, does some distribution have spaces in ansible_distribution or ansible_distribution_version or ansible_distribution_major_version or ansible_os_family ?

Reply:
Yes, see (vars/openSUSE-Leap-15.yml)[https://github.com/chiqomar/timesync/blob/adding-suse-ubuntu-support/vars/openSUSE-Leap-15.yml]